testsuite/MDAnalysisTests/coordinates/test_crd.py

```diff
@@ -40,7 +42,8 @@ def u(self):
 
     @pytest.fixture()
     def outfile(self, tmpdir):
-        return str(tmpdir) + '/out.crd'
+        #return str(tmpdir) + '/out.crd'
+        return os.path.join(str(tmpdir), 'test.crd')
```

Comment:
why this change?  iirc `tmpdir` is a `Path` object so should work ok as is?

Reply:
From what I understood from @tylerjereddy 's comment, using this construct is preferred since it does not leave artifacts behind (see full discussion at: https://github.com/MDAnalysis/mdanalysis/pull/3635#discussion_r851043069

Reply:
This was indeed what @tylerjereddy suggested.

Maybe @richardjgowers was thinking to use
```python
return str(tmpdir / 'test.crd')
```
We do need a `str` for Universe as we cannot use py.path or pathfile objects yet.

By the way, please remove commented out code.